src/inference/src/ie_core.cpp

```diff
@@ -851,7 +851,14 @@ class CoreImpl : public ie::ICore, public std::enable_shared_from_this<ie::ICore
                                                 const std::map<std::string, std::string>& config,
                                                 const std::function<void(const CNNNetwork&)>& val = nullptr) override {
         OV_ITT_SCOPE(FIRST_INFERENCE, ie::itt::domains::IE_LT, "Core::LoadNetwork::Path");
-        auto parsed = parseDeviceNameIntoConfig(deviceName, config);
+        // disable auto-batch related when loading from modelPath
+        std::string device = deviceName;
+        std::map<std::string, std::string> config_with_batch = config;
+        CleanUpProperties(device, config_with_batch, ov::auto_batch_timeout);
+        CleanUpProperties(device, config_with_batch, ov::hint::allow_auto_batching);
+        // TODO: align apply-auto-batching logic with load through network
+        coreConfig.flag_allow_auto_batching = false;
```

Comment:
once user call load_model with filename, this line globally disables auto-batching which is incorrect.

Reply:
will reimplement as discussed offline